internal/services/applications/parse/owner.go

```diff
@@ -76,3 +76,17 @@ func (id OwnerId) Segments() []resourceids.Segment {
 func (id OwnerId) String() string {
 	return fmt.Sprintf("Application Owner (Application ID: %q, Owner ID: %q)", id.ApplicationId, id.OwnerId)
 }
+
+func (id OwnerId) FromParseResult(input resourceids.ParseResult) error {
```

Comment:
This'll need to be a pointer:

```suggestion
func (id *OwnerId) FromParseResult(input resourceids.ParseResult) error {
```